test/e2e/manifests/pkg/externally-managed-service-account/setup/provider.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: pkg.crossplane.io/v1
 kind: Provider
 metadata:
-  name: provider-nop
+  name: provider-nop-pkg-external-sa
```

Comment:
oh nice, i've wondered just recently when paying more attention to e2e failures if conflicts across multiple test cases installing the same thing could be causing some flakes...